docs/source/model_doc/luke.rst

```diff
@@ -78,34 +78,32 @@ Example:
 
     >>> model = LukeModel.from_pretrained("studio-ousia/luke-base")
     >>> tokenizer = LukeTokenizer.from_pretrained("studio-ousia/luke-base")
 
     # Example 1: Computing the contextualized entity representation corresponding to the entity mention "Beyoncé"
     >>> text = "Beyoncé lives in New York."
     >>> entity_spans = [(0, 7)]  # character-based entity span corresponding to "Beyoncé"
-    >>> encoding = tokenizer(text, entity_spans=entity_spans, add_prefix_space=True, return_tensors="pt")
-    >>> outputs = model(**encoding)
+    >>> inputs = tokenizer(text, entity_spans=entity_spans, add_prefix_space=True, return_tensors="pt")
+    >>> outputs = model(**inputs)
     >>> word_last_hidden_state = outputs.last_hidden_state
     >>> entity_last_hidden_state = outputs.entity_last_hidden_state
 
     # Example 2: Inputting Wikipedia entities to obtain enriched contextualized representations
-    >>> text = "Beyoncé lives in New York."
     >>> entities = ["Beyoncé", "New York City"]  # Wikipedia entity titles corresponding to the entity mentions "Beyoncé" and "New York"
     >>> entity_spans = [(0, 7), (17, 25)]  # character-based entity spans corresponding to "Beyoncé" and "New York"
-    >>> encoding = tokenizer(text, entities=entities, entity_spans=entity_spans, add_prefix_space=True, return_tensors="pt")
-    >>> outputs = model(**encoding)
+    >>> inputs = tokenizer(text, entities=entities, entity_spans=entity_spans, add_prefix_space=True, return_tensors="pt")
+    >>> outputs = model(**inputs)
     >>> word_last_hidden_state = outputs.last_hidden_state
     >>> entity_last_hidden_state = outputs.entity_last_hidden_state
 
-    # Example 3: Classifying an entity using LukeForEntityClassification head model
-    >>> model = LukeForEntityClassification.from_pretrained("studio-ousia/luke-base")
-    # Instantiate LukeTokenizer with specifying task="entity_classification" to create input for LukeForEntityClassification
-    >>> tokenizer = LukeTokenizer.from_pretrained("studio-ousia/luke-base", task="entity_classification")
-    >>> text = "Beyoncé lives in New York."
-    >>> entity_spans = [(0, 7)]  # character-based entity span corresponding to "Beyoncé"
+    # Example 3: Classifying the relationship between two entities using LukeForEntityPairClassification head model
+    >>> model = LukeForEntityPairClassification.from_pretrained("studio-ousia/luke-large-finetuned-tacred")
+    # Instantiate LukeTokenizer with specifying task="entity_pair_classification" to create input for LukeForEntityPairClassification
+    >>> tokenizer = LukeTokenizer.from_pretrained("studio-ousia/luke-large-finetuned-tacred", task="entity_pair_classification")
+    >>> entity_spans = [(0, 7), (17, 25)]  # character-based entity spans corresponding to "Beyoncé" and "New York"
     >>> inputs = tokenizer(text, entity_spans=entity_spans, return_tensors="pt")
     >>> outputs = model(**inputs)
     >>> logits = outputs.logits
```

Comment:
It would be cool to also add

```
predicted_class_idx = logits.argmax(-1)
print("Predicted class:", model.config.id2label[predicted_class_idx])
```
for each of the fine-tuned head models. In that way, we are sure that the 

(I don't know what the different relationships are in TACRED, perhaps we could take a more appropriate sentence from this dataset)

Reply:
I'll do it :)